app/assets/javascripts/discourse/app/components/uppy-backup-uploader.js

```diff
@@ -1,13 +1,29 @@
 import Component from "@ember/component";
+import { not } from "@ember/object/computed";
 import I18n from "I18n";
 import UppyUploadMixin from "discourse/mixins/uppy-upload";
 import discourseComputed from "discourse-common/utils/decorators";
 
 export default Component.extend(UppyUploadMixin, {
+  id: "uppy-backup-uploader",
   tagName: "span",
   type: "backup",
-  useMultipartUploadsIfAvailable: true,
+
   uploadRootPath: "/admin/backups",
+  uploadUrl: "/admin/backups/upload",
+
+  // TODO (martin) Add functionality to make this usable _without_ multipart
+  // uploads, direct to S3, which needs to call get-presigned-put on the
+  // BackupsController (which extends ExternalUploadHelpers) rather than
+  // the old create_upload_url route. The two are functionally equivalent;
+  // they both generate a presigned PUT url for the upload to S3, and do
+  // the whole thing in one request rather than multipart.
+
+  // direct s3 backups
+  useMultipartUploadsIfAvailable: not("localBackupStorage"),
+
+  // local backups
+  useChunkedUploads: not("useMultipartUploadsIfAvailable"),
```

Comment:
useChunkedUploads is not "useMultipartUploads" so it's not not "localBackupStorage"?

Reply:
Changed to `alias(localBackupStorage)` but yeah will change again to address that TODO above